addons/spreadsheet_dashboard/models/spreadsheet_dashboard.py

```diff
@@ -30,6 +38,14 @@ def get_readonly_dashboard(self):
             'default_currency': default_currency,
         }
 
+    def dashboard_is_empty(self, revisions=[]):
+        if len(revisions) > 0:
+            return False
+        return (
+            any(self.env[model.model].search_count([], limit=1) == 0 for model in self.main_data_model_ids)
+            and self.env["spreadsheet.revision"].with_context(active_test=False).search_count([("res_id", "=", self.id), ("res_model", "=", self._name)], limit=1) == 0
```

Comment:
The model spreadsheet.revision doesn't exist in community ;)

Reply:
bien vu, merci :heart: